.storybook/boilerPlate.js

```diff
@@ -0,0 +1,20 @@
+import { ThemeProvider } from '@material-ui/core/styles'
+import { action } from '@storybook/addon-actions'
+import catsTheme, { tabForTeamSeasTheme } from 'src/themes/theme'
+import { withGlobals } from '@luigiminardim/storybook-addon-globals-controls'
+import CssBaseline from '@material-ui/core/CssBaseline'
+
+export const mobile = (template) => (props) =>
```

Comment:
@kmjennison storybook deprecated the ability to dynamically create stories so I went with an hoc style way of composing responsive and themed stories for homepage components.  The Standard story in each component will use the global controls so that you can switch back and forth between themes (and potentially widths) more easily in storybook, then I think we can follow a consistent named story convention (IE: MobileCats,FullWidthSeas) for chromatic.  You can see chromatic for an example